CodecException when using "External roles and attributes mapping"

I want to use an external mapper (https://akhq.io/docs/configuration/authentifications/external.html) for granting granular access to topics and consumergroups.
Therefore, I have created a small application which returns the required JSON with the mapping.

Unfortunately I get the following exception:
```
2023-10-18 13:42:05,992 ERROR Group-1-13 .m.r.i.RecoveryInterceptor Type [org.akhq.security.claim.RestApiClaimProvider$Intercepted] executed with error: Client 'http://localhost:8090/get-roles-and-attributes': Error decoding HTTP response body: Cannot decodebytes with value [{"groups":{"Administrator":[{"role":"Administrator","patterns":[".*"],"clusters":[".*"]}],"AuthenticatedUser":[{"role":"AuthenticatedUser","patterns":[".*"],"clusters":[".*"]}],"principal1_ReadOnly":[{"role":"principal_ReadOnly","patterns":["principal1-.*"],"clusters":[".*"]}]}}] to type: ClaimResponse
io.micronaut.http.client.exceptions.HttpClientResponseException: Client 'http://localhost:8090/get-roles-and-attributes': Error decoding HTTP response body: Cannot decode bytes with value [{"groups":{"Administrator":[{"role":"Administrator","patterns":[".*"],"clusters":[".*"]}],"AuthenticatedUser":[{"role":"AuthenticatedUser","patterns":[".*"],"clusters":[".*"]}],"principal1_ReadOnly":[{"role":"principal_ReadOnly","patterns":["principal1-.*"],"clusters":[".*"]}]}}] to type: ClaimResponse
at io.micronaut.http.client.netty.DefaultHttpClient$FullHttpResponseHandler.makeNormalBodyParseError(DefaultHttpClient.java:2268)
...
... 46 common frames omitted
Caused by: io.micronaut.http.codec.CodecException: Cannot decode bytes with value [{"groups":{"Administrator":[{"role":"Administrator","patterns":[".*"],"clusters":[".*"]}],"AuthenticatedUser":[{"role":"AuthenticatedUser","patterns":[".*"],"clusters":[".*"]}],"principal1_ReadOnly":[{"role":"principal_ReadOnly","patterns":["principal1-.*"],"clusters":[".*"]}]}}] to type: ClaimResponse
at io.micronaut.runtime.http.codec.TextPlainCodec.lambda$decode$1(TextPlainCodec.java:113)
at java.base/java.util.Optional.orElseThrow(Unknown Source)
at io.micronaut.runtime.http.codec.TextPlainCodec.decode(TextPlainCodec.java:113)
at io.micronaut.http.client.netty.FullNettyClientHttpResponse.convertByteBuf(FullNettyClientHttpResponse.java:279)
at io.micronaut.http.client.netty.FullNettyClientHttpResponse.lambda$getBody$1(FullNettyClientHttpResponse.java:217)
at java.base/java.util.HashMap.computeIfAbsent(Unknown Source)
at io.micronaut.http.client.netty.FullNettyClientHttpResponse.getBody(FullNettyClientHttpResponse.java:191)
2023-10-18 13:42:05,993 INFO  Group-1-13 org.akhq.log.access        [Date: 2023-10-18T13:42:05.376225Z] [Duration: 617 ms] [Url: GET /oauth/callback/oidc] [Status: 303] [Ip: /127.0.0.1] [User: Anonymous]
at io.micronaut.http.client.netty.FullNettyClientHttpResponse.<init>(FullNettyClientHttpResponse.java:110)
at io.micronaut.http.client.netty.DefaultHttpClient$FullHttpResponseHandler.buildResponse(DefaultHttpClient.java:2186)
... 46 common frames omitted
2023-10-18 13:42:05,992 WARN  Group-1-13 l.LoginFailedEventListener Login failed reason CUSTOM, username unknown, message Exception from ClaimProvider org.akhq.security.claim.RestApiClaimProvider$Intercepted: Client 'http://localhost:8090/get-roles-and-attributes': Error decoding HTTP response body: Cannot decode bytes with value [{"groups":{"Administrator":[{"role":"Administrator","patterns":[".*"],"clusters":[".*"]}],"AuthenticatedUser":[{"role":"AuthenticatedUser","patterns":[".*"],"clusters":[".*"]}],"principal1_ReadOnly":[{"role":"principal_ReadOnly","patterns":["principal1-.*"],"clusters":[".*"]}]}}] to type: ClaimResponse
```
I have tested with versions `0,24` and `dev` (I saw a note about a code merge to branch `dev` with a modified implementation). As far as I can see, the generated mapping looks fine.

Is the new functionality with fine-grained access already part of a current Docker image?

--- a/docs/docs/configuration/authentifications/external.md
+++ b/docs/docs/configuration/authentifications/external.md
@@ -76,6 +76,9 @@ and expect the following JSON as response :
   }
 }
 ````
+::: warning
+The response must contain the `Content-Type: application/json` header to prevent any issue when reading the response.
+:::
 
 **Groovy API**
 ````yaml

